kafka_consumer/tests/python_client/test_unit.py

```diff
@@ -24,27 +25,20 @@ def test_gssapi(kafka_instance, dd_run_check, check):
     # assert the check doesn't fail with:
     # Exception: Could not find main GSSAPI shared library.
     with pytest.raises(Exception, match='check_version'):
-        dd_run_check(check(instance))
+        dd_run_check(check(instance, client=KafkaPythonClient))
 
 
 def test_tls_config_ok(check, kafka_instance_tls):
     with mock.patch('datadog_checks.base.utils.tls.ssl') as ssl:
-        with mock.patch('kafka.KafkaAdminClient') as kafka_admin_client:
+        # mock TLS context
+        tls_context = mock.MagicMock()
+        ssl.SSLContext.return_value = tls_context
 
-            # mock Kafka Client
-            kafka_admin_client.return_value = mock.MagicMock()
```

Comment:
The tls_context is no longer created with the admin client initialization so this mock is no longer needed.